data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/DefaultLogstashPluginAttributesMapperTest.java

```diff
@@ -114,4 +115,20 @@ void mapAttributes_sets_additional_attributes_to_those_values() {
         assertThat(actualPluginSettings, hasKey(additionalAttributeName));
         assertThat(actualPluginSettings.get(additionalAttributeName), equalTo(additionalAttributeValue));
     }
+
+    @Test
+    void mapAttributes_with_negation_expression_negates_boolean_value() {
+        final String dataPrepperAttribute = "!".concat(UUID.randomUUID().toString());
+        boolean value = true;
```

Comment:
For boolean tests, we should use parameterized tests. It helps ensure that the code is not written to favor one boolean value over another.

```
@ParameterizedTest
@ValuesSource(booleans = { true, false } )
void mapAttributes_with_negation_expression_negates_boolean_value(boolean inputValue)
```

I used `inputValue` instead of `value` for clarity, and recommend a similar change as well.